Changes for 1st release of the integrated MetaOS (#11)
fluidos-project/quick-start-guide: README.md

> - A component that can consist of either a single device or a set of devices, primarily serving as a representation of a Kubernetes node. It is managed by a singular Kubernetes Control Plane. For additional information, kindly consult the [FLUIDOS node repository](https://github.com/fluidos-project/node)

Reviewer: ```suggestion
- A component that can consist of either a single device or a set of devices, primarily serving as a representation of a Kubernetes cluster. It is managed by a single Kubernetes Control Plane. For additional information, kindly consult the [FLUIDOS node repository](https://github.com/fluidos-project/node)
```

> - FLUIDOS at the edge: a minimal architecture for running the FLUIDOS components at the edge of the network on some STM boards, leveraging KubeEdge. For additional information, kindly consult the [FLUIDOS at the Edge repository](https://github.com/fluidos-project/fluidos-edge)

Reviewer: ```suggestion
- FLUIDOS at the edge: a minimal architecture for running the FLUIDOS components at the edge of the network on some STM boards, leveraging KubeEdge. For additional information, check at the [FLUIDOS at the Edge repository](https://github.com/fluidos-project/fluidos-edge).
```

>     The project itself is extensible, allowing the definition of specific models, or rule/heuristics, for the orchestration of the deployed workloads.

Reviewer: ```suggestion
    The project itself is extensible, allowing the definition of specific models, or rule/heuristics, for the orchestration of the deployed workloads. For additional information, check at the [FLUIDOS model-based Meta Orchestration repository](https://github.com/fluidos-project/fluidos-modelbased-metaorchestrator/).
```

>     For additional information, kindly consult the [FLUIDOS model-based Meta Orchestration repository](https://github.com/fluidos-project/fluidos-modelbased-metaorchestrator/).

Reviewer: ```suggestion
```

>     Note that the pluging also allows interaction with the MSPL-based meta-orchestrator, thus providing a developer a single tool for transparently interacting with the FLUIDOS components.

Reviewer: ```suggestion
    Note that the pluging also allows interaction with the MSPL-based meta-orchestrator, thus providing a developer a single tool for transparently interacting with the FLUIDOS components. For additional information, check at the [kubectl FLUIDOS plugin](https://github.com/fluidos-project/kubectl-fluidos-plugin/).
```

    For additional information, kindly consult the [kubectl FLUIDOS plugin](https://github.com/fluidos-project/kubectl-fluidos-plugin/).

- [kubectl FLUIDOS plugin 1st Release](https://github.com/fluidos-project/kubectl-fluidos-plugin/releases/tag/0.0.1)

</details>

<details>
    <summary> FLUIDOS Identity Management</summary>

- FLUIDOS Identity Management:
    This component consists in an Aries agent modified for use dp-abc cryptography combined with Hyperledger Fabric as VDR to provide a very powerful interface when working with DID's, issuing VCredentials or using smart contracts. 

>     These tools will help to secure any scenario that may occur in FLUIDOS and bring us an API with which we can, among other things; issue VCredentials, create DIDs for FLUIDS entities, create Verifiable Presentations from VCredentials and verify said VCredentials/VPresentations.For additional information, kindly consult the [Fluidos Identity Management Aries Framework](https://github.com/fluidos-project/idm-fluidos-aries-framework-go).

Reviewer: ```suggestion
    These tools will help to secure any scenario that may occur in FLUIDOS and bring us an API with which we can, among other things; issue VCredentials, create DIDs for FLUIDS entities, create Verifiable Presentations from VCredentials and verify said VCredentials/VPresentations. For additional information, check at the [Fluidos Identity Management Aries Framework](https://github.com/fluidos-project/idm-fluidos-aries-framework-go).
```
